content/src/Server.ts

```diff
@@ -49,7 +49,9 @@ export class Server {
 
     this.app = express()
 
-    this.metricsServer = initializeMetricsServer(this.app, metricsComponent)
+    if (env.getConfig(EnvironmentConfig.METRICS)) {
```

Comment:
I think that we are deprecating that config, as metrics will be always enabled. Can we make something only for running tests?